### DISCUSSION TITLE
ProjectOST Discussions!

### DISCUSSION THREAD

**Discussion by ColtonKeeley** [Tsaryii/ProjectOST#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Submit Bug Fixes or Suggestions

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GetPine** [comment]
Love to see so much quality work going into a free open source project like this - something the UEFN community could learn from for sure!

**Reply by ColtonKeeley** [reply]
Thanks! <3

**Comment by simnJS** [comment]
Good project, but It's a very bad idea for the UEFN Ecosystem, a lot of trash tycoon will spawn soon.

**Reply by Seniorfrito** [reply]
100% disagree. We already have a lot of trash tycoons. A polished and easy to use/understand template like ProjectOST can spawn a lot of way better tycoons.

**Comment by Atayeu** [comment]
Hey, dropped a comment into your discord about the guide / setup instructions you have on the repo front and the wiki pages. I'm willing to redo the wiki pages to fix some issues and make it a bit more elaborate and easier to understand/follow along with. Let me know if I can help!

**Comment by Zibgame** [comment]
Hello
Can you explain how the convoyer work in your script, and how to make them dropper? No one wants to help me.
